cl/free_look_email/app.py

```diff
@@ -0,0 +1,149 @@
+import json
+import requests
+from requests.exceptions import ConnectionError, HTTPError, Timeout
+from humps import decamelize
+import time
+from functools import wraps
+
+# TODO - It would be preferable to have this in a separate decorators package nearby. This currently has problems with the tests. Will research.
```

Comment:
Just commenting so we have a todo item here before this merge, unless you want to merge it and fix later, but now seems like the right time for this kind of issue.

Reply:
Yes, I was banging my head against this one a bit. I can take another look as you go through the final review.